website/docs/tutorials/structured_config/4_defaults.md

```diff
@@ -68,14 +68,37 @@ db:
   ...
 ```
 
-#### A Note about composition order
-When Hydra composes a config, by default - any values defined in the config are overriding the values coming from configs specified in the Defaults List.
-If you specify a value, Hydra 1.1 will warn that you need to specify `_self_` in the Defaults List to indicate the desired composition order (The above example does not specify a value so there is no warning).
-In cases similar to this one, you will typically want to insert `_self_` as the first item in the Defaults List. 
+### A Note about composition order
+The default composition order in Hydra is that values defined in a config are merged into values introduced from configs in the Defaults List - or in other words - overriding them.
```

Comment:
```suggestion
Hydra's default behavior is to override values in the Defaults List with values subsequently defined in the same config.
```

Reply:
The values are not in the defaults list. I know you are trying to simplify but I want to be accurate.

Reply:
how about
> Hydra's default behavior is to override values introduced by the Defaults List with values subsequently defined in the same config.

Reply:
too convoluted.

Reply:
nit:

To someone that knows hydra only as userr, this seems more clear to me (but I 'm probably missing something)

>  The default behavior in Hydra is for values defined in a config to override values from configs in the Defaults List.